Podfile

```diff
@@ -34,12 +34,13 @@ end
 
 def wordpress_ui
     ## for production:
-    pod 'WordPressUI', '~> 1.5.2'
+    pod 'WordPressUI', '~> 1.5.3-beta.1'
 
     ## for development:
     #pod 'WordPressUI', :path => '../WordPressUI-iOS'
     ## while PR is in review:
     #pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS', :branch => ''
+    #pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS', :commit => '71f32a3300b4c630b41ba7ae8101896f9d297606'
```

Comment:
Since this is already commented out, do we need to push it?

Reply:
This is for convenience during development only.  It allows quickly replacing the commit number and enabling this line rather than having to look up the command formatting.